components/ingest-service/backend/elastic/elastic.go

```diff
@@ -50,9 +50,14 @@ func (es *Backend) UpdateProjectTags(ctx context.Context, projectTaggingRules ma
 		return []string{}, errors.Wrap(err, "Failed to start Elasticsearch Node project tags update")
 	}
 
-	logrus.Debugf("Started Project rule updates with node job ID: %q", esNodeJobID)
+	esActionJobID, err := es.UpdateActionProjectTags(ctx, projectTaggingRules)
```

Comment:
am i reading right that this was the only code change actually needed for this?

Reply:
@vjeffrey That is correct! 😄

Reply:
nice!!!